ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -82,7 +91,30 @@ public class BottomSheetViewController: UIViewController {
 
         updateResizingHandleViewAccessibility()
     }
+    
+    public override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        if animator != nil {
+            // panning started, don't update the height
+            return
+        }
+        updateHeight()
+    }
 
+// MARK: private helpers
+    
+    private func updateHeight(){
+        // update the view layout for collapsed mode
+        if currentState == .collapse
+        {
+            if let window = self.view.window {
+                let windowFrame = window.frame
+                view.frame = CGRect(x: 0, y: (windowFrame.height - collapsedHeight), width: windowFrame.width, height: collapsedHeight)
```

Comment:
can we refactor this with the setUPAnimator() as well so there is one function calling for both places for consistency?

Reply:
@harrieshin 

didn't get the point, in setupAnimator we are using animation and animating it to target position / frame. 
updateHeight we are setting the frame.

I can add one private function which will provide frame for given window and 
can call this method in .expand case of setupAnimator and updateHeight ?

Reply:
it could be a simple function like  private func targetCollapseFrame(with window: UIWindow)->CGRect